integration_test/utils/scenario_utils_mixin.dart

```diff
@@ -142,34 +142,38 @@ mixin ScenarioUtilsMixin {
         filePath: path,
         fileSize: await file.length(),
       );
-      final uploadUri = mailboxDashBoardController.sessionCurrent!.getUploadUri(
-        mailboxDashBoardController.accountId.value!,
-        jmapUrl: mailboxDashBoardController.dynamicUrlInterceptors.jmapUrl,
-      );
-      
-      final uploadAttachmentInteractor = Get.find<UploadAttachmentInteractor>();
-      final uploadAttachmentState = await uploadAttachmentInteractor
-        .execute(fileInfo, uploadUri)
-        .last;
-      final attachment = await uploadAttachmentState.fold(
-        (failure) => null,
-        (success) async {
-          if (success is UploadAttachmentSuccess) {
-            final uploadAttachment = await success.uploadAttachment.progressState.last;
-            return uploadAttachment.fold(
-              (failure) => null,
-              (success) {
-                if (success is! SuccessAttachmentUploadState) return null;
-
-                return success.attachment;
-              },
-            );
-          }
-        },
-      );
-      if (attachment != null) {
-        attachments.add(attachment);
-      }
+
+      try {
+        final uploadUri = mailboxDashBoardController.sessionCurrent!.getUploadUri(
+          mailboxDashBoardController.accountId.value!,
+          jmapUrl: mailboxDashBoardController.dynamicUrlInterceptors.jmapUrl,
+        );
+
+        final uploadAttachmentInteractor = Get.find<UploadAttachmentInteractor>();
+        final uploadAttachmentState = await uploadAttachmentInteractor
+          .execute(fileInfo, uploadUri)
+          .last;
+        final attachment = await uploadAttachmentState.fold(
+          (failure) => null,
+          (success) async {
+            if (success is UploadAttachmentSuccess) {
+              final uploadAttachment = await success.uploadAttachment.progressState.last;
+              return uploadAttachment.fold(
+                (failure) => null,
+                (success) {
+                  if (success is! SuccessAttachmentUploadState) return null;
+
+                  return success.attachment;
+                },
+              );
+            }
+          },
+        );
+        if (attachment != null) {
+          attachments.add(attachment);
+        }
+      } catch (e) {
+        mailboxDashBoardController.handleFailureViewState(UploadAttachmentFailure(e, fileInfo));}
```

Comment:
This is a function that handles only the upload result to support email creation in `integration test`. So we don't need to handle the failure value. Please delete it.